src/current/v23.2/ui-network-latency-page.md

```diff
@@ -1,29 +1,34 @@
 ---
-title: Network Latency Page
+title: Network Page
 summary: The Network Latency page displays round-trip latencies between all nodes in your cluster.
 toc: true
-docs_area: reference.db_console
 ---
 
-The **Network Latency** page displays round-trip latencies between all nodes in your cluster. Latency is the time required to transmit a packet across a network, and is highly dependent on your network topology. Use this page to determine whether your latency is appropriate for your [topology pattern]({% link {{ page.version.version }}/topology-patterns.md %}), or to identify nodes with unexpected latencies.
+The **Network** page displays round-trip latencies between all nodes in your cluster. Latency is the time required to transmit a packet across a network, and is highly dependent on your network topology. Use this page to determine whether your latency is appropriate for your [topology pattern]({% link {{ page.version.version }}/topology-patterns.md %}), or to identify nodes with unexpected latencies or connectivity issues.
```

Comment:
kept "between" because it is used in [this context](https://www.grammarly.com/blog/between-among/#:~:text=In%20fact%2C%20you%20can%20use%20between%20for%20any%20number%20of%20elements%2C%20as%20long%20as%20all%20the%20elements%20are%20separate%20and%20distinct.%20According%20to%20The%20Chicago%20Manual%20of%20Style%2C%20you%20can%20even%20use%20between%20when%20%E2%80%9Cmultiple%20one%2Dto%2Done%20relationships%20are%20understood%20from%20the%20context%E2%80%9D%3A).

updated as suggested: "This represents" and "and".